README.md

```diff
@@ -1,6 +1,129 @@
-gatling-jenkins-plugin
+Gatling Jenkins Plugin
 ======================
 
-Jenkins plugin for Gatling
+
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/gatling.svg)](https://plugins.jenkins.io/gatling)
+[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/gatling.svg?color=blue)](https://stats.jenkins.io/pluginversions/gatling.html)
+[![Jenkins CI](https://ci.jenkins.io/buildStatus/icon?job=Plugins/gatling-plugin/master)](https://ci.jenkins.io/blue/organizations/jenkins/Plugins%2Fgatling-plugin/activity/)
+
+This plugin integrates [Gatling](http://gatling.io/), an Open Source
+stress tool, with Jenkins.
+
+## Features
+
+This plugin allows you to : 
+
+- Keep track of a Gatling simulation, providing performance trends across builds
+- Publish detailed reports for each build
+
+## Changelog
+
+**A [XSS vulnerability](https://jenkins.io/security/advisory/2020-04-07/#SECURITY-1633) was fixed in the version 1.3.0 of the plugin, be careful when using older version of the plugin.**
+
+![(warning)](https://wiki.jenkins.io/s/en_GB/8100/5084f018d64a97dc638ca9a178856f851ea353ff/_/images/icons/emoticons/warning.svg)
+ If you're coming from 1.0.0, make sure "Enable simulation tracking"
+checkbox is checked in your build configuration and if not, check it.
+
+This is an issue with configuration changes between 1.0.0 and 1.0.1
+and doesn't affect fresh 1.0.1 installations.
+
+- 1.3.0
+    -   Fix [XSS vulnerability](https://jenkins.io/security/advisory/2020-04-07/#SECURITY-1633)
```

Comment:
Reports can no longer be displayed directly online from Jenkins but have to be downloaded locally.